tests/console/validate_encrypted_partition_not_activated.pm

```diff
@@ -19,10 +19,11 @@ use base "installbasetest";
 use scheduler 'get_test_suite_data';
 use testapi;
 use validate_encrypt_utils;
+use Utils::Backends 'use_ssh_serial_console';
 
 sub run {
     my $test_data = get_test_suite_data();
-    select_console 'install-shell';
+    check_var('BACKEND', 'pvm_hmc') ? use_ssh_serial_console : select_console 'install-shell';
```

Comment:
I guess it's same story as we have reported in https://progress.opensuse.org/issues/69406
Could you please add reference to this code to the ticket, so we don't forget to fix it. Or, as an alternative, we can skip enabling this module on powerVM for now and act on it, once issue is solved. WDYT?

Reply:
The console switching works with that solution, but as I had pointed out in https://progress.opensuse.org/issues/69406; noz there is the problem that it seems like disk has not yet finished to be encrypted, as explained in comment 7: https://progress.opensuse.org/issues/69406#note-7. So the info is already in the ticket, but I could add it to the description.

Reply:
Regarding the false positive, I think I could just submit a follow-up ticket and use it as-is for now, as I have already another ticket in my queue.

Reply:
That's very strange as we are supposed not to activate anything, so there should be no action triggered by the installer. So I'm not sure if I understand the point. As for the false positive part, I don't think that we want to have flaky test to review.

Reply:
> s very strange as we are supposed not to activate anything
The comment #7 is for cryptlvm+activate.